opencti-platform/opencti-graphql/src/modules/outcome/outcome.graphql

```diff
@@ -0,0 +1,81 @@
+enum OutcomeFilter {
+    created
+}
+input OutcomeFiltering {
+    key: [OutcomeFilter!]!
+    values: [String]
+    operator: String
+    filterMode: FilterMode
+}
+enum OutcomeOrdering {
+    name
+    created
+}
+
+type OutcomeParameter {
+    key: String
+    value: String
+}
+
+type OutcomeConnector {
+    id: ID! # internal_id
+    name: String!
+    built_in: Boolean
+    connector_type: String
+    connector_schema: String!
+    connector_schema_ui: String
+}
+
+type Outcome implements InternalObject & BasicObject {
+    id: ID! # internal_id
+    standard_id: String!
+    entity_type: String!
+    parent_types: [String]!
+    created: DateTime
+    modified: DateTime
+    name: String!
+    description: String
+    outcome_connector: OutcomeConnector!
+    outcome_connector_id: String!
+    outcome_configuration: String!
+    authorized_members: [MemberAccess!] @auth(for: [SETTINGS])
```

Comment:
could it be named "authorizedMembers" ? not to be confused with the stored field "authorized_members". Same for "outcomeConnector"